### ISSUE TITLE
Firefox Webdriver is not working with current Aurora

### ISSUE THREAD

**Issue by roidelapluie** [SeleniumHQ/selenium#1423]
```
======================================================================
ERROR: testLoginAsAdminUser (app.tests.LoginTests)
Test login as an admin user
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/AlexandreDecan/Lexpage/app/tests.py", line 281, in testLoginAsAdminUser
    self.login('admin', 'admin')
  File "/home/travis/build/AlexandreDecan/Lexpage/app/tests_helpers.py", line 151, in login
    self.fill_in_login_form(username, password, webdriver=webdriver)
  File "/home/travis/build/AlexandreDecan/Lexpage/app/tests_helpers.py", line 139, in fill_in_login_form
    username_input.send_keys(username)
  File "/home/travis/virtualenv/python3.4.3/lib/python3.4/site-packages/selenium/webdriver/remote/webelement.py", line 328, in send_keys
    self._execute(Command.SEND_KEYS_TO_ELEMENT, {'value': typing})
  File "/home/travis/virtualenv/python3.4.3/lib/python3.4/site-packages/selenium/webdriver/remote/webelement.py", line 454, in _execute
    return self._parent.execute(command, params)
  File "/home/travis/virtualenv/python3.4.3/lib/python3.4/site-packages/selenium/webdriver/remote/webdriver.py", line 201, in execute
    self.error_handler.check_response(response)
  File "/home/travis/virtualenv/python3.4.3/lib/python3.4/site-packages/selenium/webdriver/remote/errorhandler.py", line 181, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: cross-process JS call failed

Stacktrace:
    at Utils.keyEvent (file:///tmp/tmp5ttw80qf/extensions/fxdriver@googlecode.com/components/command-processor.js:9074)
    at Utils.type (file:///tmp/tmp5ttw80qf/extensions/fxdriver@googlecode.com/components/command-processor.js:9050)
    at WebElement.sendKeysToElement/< (file:///tmp/tmp5ttw80qf/extensions/fxdriver@googlecode.com/components/command-processor.js:11999)
    at fxdriver.Timer.prototype.setTimeout/<.notify (file:///tmp/tmp5ttw80qf/extensions/fxdriver@googlecode.com/components/driver-component.js:621)
```

https://travis-ci.org/AlexandreDecan/Lexpage/jobs/99023195

With Firefox aurora, send_keys makes Firefox crash. Do you feel that it is a problem with the driver or just firefox aurora?

Exception is `Message: cross-process JS call failed`.

Driver capabilities:

```
version : 45.0a2
browserName : firefox
applicationCacheEnabled : True
platform : Linux
takesScreenshot : True
acceptSslCerts : True
webStorageEnabled : True
handlesAlerts : True
locationContextEnabled : True
rotatable : False
javascriptEnabled : True
cssSelectorsEnabled : True
databaseEnabled : True
nativeEvents : False
```

**Comment by lukeis** [comment]
This is a problem with the current implementation and future versions of Firefox. Most of the way the webdriver extension performs it's actions is no longer compatible. 

Without an extensive rewrite, the driver implementation here will no longer work with FF45+

The only way forward is to start using marionette that is being implemented by Mozilla themselves. But sadly it is not going to see a 1.0 release until well after FF45.

**Comment by cgoldberg** [comment]
yikes!

**Comment by barancev** [comment]
This exact issue is fixed by commit 2883bafa12c3e872baf2a64b917a1b0b77b40ccd

Yet there are many other regression test failures in Firefox 45, so I'll keep this issue open to track the progress.

**Comment by barancev** [comment]
This issue was fixed long ago as stated in the previous commit. And the regression tests were fixed too.
